### PR TITLE
add new contact fields

### PR DESCRIPTION
**Pull request by jlaplante333**
add new contact fields to the model

### CODE REVIEW

**Comment by cedar-technologies** [comment]
resolve #8

File: src/app/api/business/contacts/models.py [hunk 1]
```diff
@@ -2,8 +2,78 @@
...
+
+address fields
+https://en.wikipedia.org/wiki/ISO_3166-2
+
+"""
+
+    'city': fields.city(readOnly=True, description='City of contact address'),
+    'country': fields.country(readOnly=True, description='Country of contact address'),
+    'region': fields.region(readOnly=True, description='Region of contact address(e.g. State, Province, Territory, Region, District)'), //Please read: https://en.wikipedia.org/wiki/ISO_3166-2
+    'street': fields.street(readOnly=True, description='Street of contact address'),
```
[the remark this answers is not in the document]
Comment (jlaplante333) [reply src/app/api/business/contacts/models.py]: There's only one street for a single address though, you only want to send a letter to one street address / mailing address. Alternate address can be used for a secondary address (compound address.)

File: src/app/api/business/contacts/models.py [hunk 2]
```diff
@@ -2,8 +2,78 @@
...
+'fax': fields.fax(readOnly=True, description='Fax number of contact.'),
+'opted-out-of-fax': fields.Boolean(readOnly=True, description='Indicates whether the contact prohibits receiving faxes.'),
+
+
+"""
+
+activity information
+
+"""
+'last-activity-date': fields.date(readOnly=True, description='Last date where an event was logged for a contact.'),
```
[the remark this answers is not in the document]
Comment (jlaplante333) [reply src/app/api/business/contacts/models.py]: Ok commenting out, it's in Salesforce, SAP CRM, I think it's good to know the last date it was modified or last date it was active etc

File: src/app/api/business/contacts/models.py [hunk 3]
```diff
@@ -2,8 +2,78 @@
...
+    'alternate-latitude': fields.latitude(readOnly=True, description='Alternate latitude of contact address'),
+    'alternate-longitude': fields.longitude(readOnly=True, description='Alternate longitude of contact address'),
+
+"""
+
+phone number information
+
+"""
+
+ 'phone-number': fields.Phone(readOnly=True, description='Phone Number of a contact, label should be business phone'),
```
[the remark this answers is not in the document]
Comment (jlaplante333) [reply src/app/api/business/contacts/models.py]: Ok

File: src/app/api/business/contacts/models.py [hunk 4]
```diff
@@ -2,8 +2,78 @@
...
+https://en.wikipedia.org/wiki/ISO_3166-2
+
+"""
+
+    'city': fields.city(readOnly=True, description='City of contact address'),
+    'country': fields.country(readOnly=True, description='Country of contact address'),
+    'region': fields.region(readOnly=True, description='Region of contact address(e.g. State, Province, Territory, Region, District)'), //Please read: https://en.wikipedia.org/wiki/ISO_3166-2
+    'street': fields.street(readOnly=True, description='Street of contact address'),
+    'postal-code': fields.String(readOnly=True, description='Postal Code of contact address'),
+    'latitude': fields.latitude(readOnly=True, description='Latitude of contact address'),
```
[the remark this answers is not in the document]
Comment (jlaplante333) [reply src/app/api/business/contacts/models.py]: ok

File: src/app/api/business/contacts/models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (cedar-technologies) [reply src/app/api/business/contacts/models.py]: yes we will need a mechanism for tracking that on all identities

File: src/app/api/business/contacts/models.py [hunk 5]
```diff
@@ -31,49 +31,7 @@
...
-
-
-"""
-
-activity information
-
-"""
-'last-activity-date': fields.date(readOnly=True, description='Last date where an event was logged for a contact.'),
-'last-viewed-date': fields.date(readOnly=True, description='Last date where a contact was viewed.'),
-
```
Comment (cedar-technologies) [inline src/app/api/business/contacts/models.py]: looks good now

**Pull request merged**